main.go

```diff
@@ -69,7 +74,8 @@ func main() {
 		return nil
 	}
 
-	app.Action = runAction
+	// Default to 'start' is no command is specified
+	app.Action = startCommand.Action
```

Comment:
yep we could if the long-term direction is to always require an 'action' verb.  Want me to update the docs to always show an action?